assets/dev/js/editor/document/elements/commands/empty.js

```diff
@@ -15,7 +19,7 @@ export default class extends Base {
 				type: 'remove',
 				elementType: 'section',
 				title: elementor.translate( 'all_content' ),
-				data: elementor.elements.toJSON(),
+				data: elementor.elements ? elementor.elements.toJSON() : null,
```

Comment:
for tests?

Reply:
Test pass without the if, i'm not remember why i put it there.